src/main/java/io/openliberty/tools/common/arquillian/util/HttpPortUtil.java

```diff
@@ -140,13 +148,7 @@ private static String getHttpPortFromConfigVariableXML(String configVariableXML,
             return null;
         }
 
-        // get input XML Document
-        DocumentBuilderFactory inputBuilderFactory = DocumentBuilderFactory.newInstance();
-        inputBuilderFactory.setIgnoringComments(true);
-        inputBuilderFactory.setCoalescing(true);
-        inputBuilderFactory.setIgnoringElementContentWhitespace(true);
-        inputBuilderFactory.setValidating(false);
-        DocumentBuilder inputBuilder = inputBuilderFactory.newDocumentBuilder();
+        DocumentBuilder inputBuilder = getBuilderFactory().newDocumentBuilder();
```

Comment:
There are several settings removed here that are not set in `getBuilderFactory()` (ignoring comments, ignoring whitespace, etc.). I think this could cause problems for the maven/gradle plugins that use this method.

Reply:
Added back